src/gluonnlp/data/transforms.py

```diff
@@ -742,6 +757,7 @@ def __call__(self, sample):
 
     def _tokenize(self, text):
         """Tokenizes a piece of text."""
+        text = convert_to_unicode(text)
```

Comment:
In https://github.com/dmlc/gluon-nlp/pull/464#issuecomment-449557024 we decide to not perform unicode conversion inside the tokenizer. The user needs to make sure the inputs are unicode (or he/she can call UnicodeTransform to convert all inputs). Please do not add it here

Reply:
Yep. I now get the discussion in #464, and I will remove it.